src/components/SignatureDishes/SignatureDishes.module.scss

```diff
@@ -0,0 +1,87 @@
+@import "../../styles//mediaQueries.scss";
+@import "../../styles/constants.module.scss";
+.signatureDishesLayout {
+  display: flex;
+  flex-flow: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 100%;
+  padding-left: 20px;
+  gap: 8px;
+
+  @include smallDesktopScreen {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    gap: 40px;
+    padding: 0;
+    width: $desktop-view-width;
+  }
+
+  .signatureDishesTitle {
+    font-size: 18px;
+    line-height: 35px;
+    letter-spacing: 1.25px;
+    @include smallDesktopScreen {
+      font-size: 30px;
+      text-align: center;
+    }
+  }
+  .signatureDishesContainer {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: flex-start;
+    width: 100%;
+    gap: 17.2px;
+
+    @include smallDesktopScreen {
+      display: flex;
+      flex-flow: column;
+      justify-content: center;
+      align-items: center;
+      width: 100%;
+      gap: 40px;
+    }
+    .carouselContainer {
+      width: 100%;
+      height: 100%;
+      .swiper {
+        .swiper-slide {
+          width: fit-content;
+        }
+      }
+    }
+    .desktopSignatureDishesContainer {
+      @include smallDesktopScreen {
+        display: flex;
+        flex-flow: row;
+        justify-content: space-between;
+        align-items: center;
+        overflow-x: scroll;
```

Comment:
This should be `overflow-x: auto;`
`scroll` will leave the scrollbar even if it is not needed.
`auto` will remove it if ther content is not actually overflowing.

Reply:
Same for everywhere else that you put `scroll` unintentionaly.

Reply:
@OrenMoveo